v03_pipeline/lib/annotations/sv.py

```diff
@@ -1,11 +1,216 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any
+from typing import Any
 
-# TODO: remove me once hail is used.
-if TYPE_CHECKING:
-    import hail as hl
+import hail as hl
 
+from hail_scripts.computed_fields import variant_id as expression_helpers
 
-def variant_id(ht: hl.Table, **_: Any) -> hl.Expression:
-    return ht.rsid
+from v03_pipeline.lib.annotations.enums import (
+    SV_CONSEQUENCE_RANKS,
+    SV_TYPE_DETAILS,
+    SV_TYPES,
+)
+from v03_pipeline.lib.model.definitions import ReferenceGenome
+
+BOTHSIDES_SUPPORT = 'BOTHSIDES_SUPPORT'
+CONSEQ_PREDICTED_PREFIX = 'PREDICTED_'
+NON_GENE_PREDICTIONS = {
+    'PREDICTED_INTERGENIC',
+    'PREDICTED_NONCODING_BREAKPOINT',
+    'PREDICTED_NONCODING_SPAN',
+}
+PASS = 'PASS'  # noqa: S105
+
+PREVIOUS_GENOTYPE_N_ALT_ALLELES = hl.dict(
```

Comment:
I left this logic as is, but we _could_ try to be smarter and do something like pass the previous genotype to seqr as `prev_GT` and let the seqr backend manage the complexity.